scripts/checks

```diff
@@ -11,20 +11,51 @@ fi
 
 BUILD_PKGS=$1
 
-echo "Checking Go version..."
+echo "+++ Checking Go version..."
 ver=$(go version | awk '{print $3}')
 minVer="go1.4"
 if [[ ${ver} < ${minVer} ]]; then
-    echo "Go version check failed. Expected >=${minVer} but found ${ver}"
+    echo "!!! Go version check failed. Expected >=${minVer} but found ${ver}"
     exit 1
 fi
 
-echo "Checking gofmt..."
+echo "+++ Checking gofmt..."
 fmtRes=$(gofmt -l $BUILD_PKGS)
 if [ -n "${fmtRes}" ]; then
-    echo -e "gofmt checking failed:\n${fmtRes}"
+    echo -e "!!! gofmt checking failed:\n${fmtRes}"
     exit 1
 fi
 
-echo "All checks passed!!"
+if ! go get github.com/golang/lint/golint
```

Comment:
This is done; I moved everything to a `scripts/deps` since it was easier to.

-Erik

> On May 21, 2015, at 11:07 AM, Madhav Puri <notifications@github.com> wrote:
> 
> In scripts/checks <https://github.com/contiv/netplugin/pull/75#discussion_r30829702>:
> 
> >      exit 1
> >  fi
> >  
> > -echo "All checks passed!!"
> > +if ! go get github.com/golang/lint/golint
> We can possibly move downloading golint to the 'deps' target in Makefile
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/75/files#r30829702>.
>